auth-service/src/main/resources/templates/thymeleaf/emails/user-added-to-dclg-in-access-group.html

```diff
@@ -24,16 +24,11 @@
 
 <p>
     This email is being sent to all delegated administrators and lead testers for <span th:text="${accessGroupName}">#accessGroupName#</span>.
-    Please contact <a th:href="'mailto:' + ${userEmailAddress}"><span
-    th:text="${userFullName}">#userFullName#</span></a>
-    or <a href="mailto:DELTAAdmin@levellingup.gov.uk">DELTAAdmin@levellingup.gov.uk</a> if you think that they should
-    not have access to this collection.
 </p>
 
 <p>
     If you have any questions or need help please email the DLUHC Service Desk on <a
-    href="mailto:dluhc.digital-services@levellingup.gov.uk">dluhc.digital-services@levellingup.gov.uk</a> or call
-    0203 829 0743.
```

Comment:
I'd leave the number 🤷‍♂️ 

It appears in a lot of places, so if we want to get rid of it, it's a slightly bigger change. I assume it goes to Exela.